doc/examples/Finite_State_Projection/Finite_State_Projection.ipynb

```diff
@@ -79,7 +79,9 @@
     "    \n",
     "    model.add_ode(state_id = f\"x_{r}_{p}\", \n",
     "                  right_hand_side=rhs,\n",
-    "                  initial_value=poisson.pmf(r, lambda_r)*poisson.pmf(p, lambda_p))\n",
+    "                  #convert from np.float to float\n",
```

Comment:
might be good to perform float conversion inside the model always.

Reply:
Marta once suggested to use pydantic here. Agree, that we should think of adding type-checks inside of the Model Editor